src/shared/component/SwaggerURLInput.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './SwaggerURLInput.scss';
+
+class SwaggerURLInput extends React.Component {
+
+  onSubmit(e) {
+    e.preventDefault();
+
+    const swaggerURL = this.swaggerURLInput.value;
```

Comment:
thank you Florie. I'll leave it unchanged for now. There will be a refactoring ticket where we refactor the code for better maintainability.